src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { LineChart, ResponsiveContainer, Line, YAxis, ReferenceLine } from 'recharts'
+import { styled } from 'rt-theme'
+import { seriesData } from '../test-resources/spotTileProps'
+
+const AnalyticsTileChartWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+`
+
+class AnalyticsTileChart extends React.Component {
+  render() {
+    const lineProps = { strokeDasharray: '4 3', stroke: '#737987', strokeOpacity: 0.9, strokeWidth: 0.8 }
```

Comment:
Haven't tested the changes myself, but just as a heads up - please make sure all colors work well with both light and dark themes

Reply:
Thanks. The colour of the line is invariant to the theme as per Zeplin design guidelines. Furthermore, I have not yet been able to manipulate the lines attributes outisde a Rechart Component.